src/docs/development/ui/widgets-intro.md

```diff
@@ -2,19 +2,23 @@
 title: Introduction to widgets
 ---
 
-{% comment %} This page needs more cleanup. API references should be accumulated
-at the end of the page to avoid repeating links. {% endcomment -%}
+{% comment %} This page needs even(more) => cleanup. API references should be accumulated
+at the floating side of the page to avoid repeating links. {% endcomment -%}
 
 {% assign api = site.api | append: '/flutter' -%}
 
-Flutter widgets are built using a modern react-style framework, which takes
-inspiration from [React](http://facebook.github.io/react). The central idea is
-that you build your UI out of widgets. Widgets describe what their view
-should look like given their current configuration and state. When a widget's
-state changes, the widget rebuilds its description, which the framework diffs
-against the previous description in order to determine the minimal changes
+`Flutter` widgets are built using a modern reactive framework (MVC+), which takes
```

Comment:
Hi @pedromorgan! I now see how many PRs you've submitted against the docs! Thanks for your interest! (I've been away for a couple of weeks.)

You would probably not believe the discussions around this phrasing. Basically, the final decision was to describe Flutter as react-style, because some folks have debated what reactive really means and what is truly reactive. In order to avoid that debate, we call ourselves react-style.